RELEASE_PROCESS.md

```diff
@@ -160,17 +163,23 @@ Note that no changes other than CHANGELOG/RELEASE_NOTES changes should land on m
 Chef / [chef/chef-server:main] habitat/build / main
 https://buildkite.com/chef/chef-chef-server-main-habitat-build
 
+==============
+This section (below) needs additional information, such as... what are we even in, a dev env, a terraform machine?  Which scenarios, configs, platforms?  How to run the supermarket test?  If it's a terraform scenario, how to activate the supermarket test when running it (I looked in the docs but didn't see anything)?  What about a step by step walkthrough as we have in sections above?
+==============
```

Comment:
https://github.com/chef/umbrella/tree/main/chef-server/scenarios/aws/supermarket-upgrade is run for every test cycle. So we do not need manual supermarket testing anymore.

Reply:
@PrajaktaPurohit Should I remove this section from the docs?

Reply:
yes!

Reply:
Done.